druid/src/util.rs

```diff
@@ -50,3 +50,28 @@ where
         }
     }
 }
+
+/// An enum for specifying whether an event was handled.
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub enum Handled {
+    /// An event was already handled, and shouldn't be propagated to other event handlers.
+    Yes,
+    /// An event has not yet been handled.
+    No,
+}
+
+impl Handled {
+    /// Has the event been handled yet?
+    pub fn is_handled(self) -> bool {
+        matches!(self, Handled::Yes)
```

Comment:
Considering that `Handled` implements `Eq`:
```suggestion
        self == Handled::Yes
```